src/main/webapp/app/applications/instance/instance.component.ts

```diff
@@ -65,6 +65,44 @@ export default class JhiInstance extends Vue {
     this.instanceModal.show();
   }
 
+  /* Modal dialog */
+  public confirmShutdown(instance: Instance): void {
+    const config = {
+      title: 'Please Confirm',
+      size: 'sm',
+      buttonSize: 'sm',
+      okVariant: 'danger',
+      okTitle: 'YES',
+      cancelTitle: 'NO',
+      footerClass: 'p-2',
+      hideHeaderClose: false,
+      centered: true
+    };
+    this.$bvModal
+      .msgBoxConfirm('Are you sure you want to shutdown the instance ?', config)
+      .then(res => {
+        if (res) {
+          this.shutdownInstance(instance);
+        }
+      })
+      .catch(err => console.warn(err));
+  }
+
+  /* shutdown an instance */
+  public shutdownInstance(instance: Instance): void {
+    this.instanceService()
+      .shutdownInstance(instance)
+      .then(() => this.$router.go(0))
```

Comment:
Supposing I'm on instance page with refresh value to 5sec.
I'm testing this and it looks like I go back to the instances page but the refresh value is gone
Maybe here, instead of going to route, just display a toast with "success" ?

Reply:
Implemented. Reloading the entire page wasn't a good idea.